docker-entrypoint.sh

```diff
@@ -36,6 +36,14 @@ build() {
   umask 0022
   svn checkout "http://svn.freetz.org/$rev" /freetz
 
+  echo "applying patches.."
+  for p in /patches/*.patch
+  do
+   [ -f "$p" ] || continue
+   echo "Apply $p"
+   patch -p1 < $p
```

Comment:
Could you quote `$p` (i.e. `patch -p1 < "$p"`) as suggested by shellcheck?

```sh
$ make lint
shellcheck *.sh

In docker-entrypoint.sh line 44:
   patch -p1 < $p
               ^-- SC2086: Double quote to prevent globbing and word splitting.

make: *** [lint] Error 1
```